commit 0be730b3ca257baecfcce52ff736b8f5ef0232eb
Author: HaseenaSainul <h.sainul@metrological.com>
Date:   Tue Feb 4 10:47:34 2020 +0530

    Compositor:Nexus: add NXSERVER find package also in the dependency of NEXUS_SERVER_EXTERNAL

diff --git a/Compositor/lib/Nexus/CMakeLists.txt b/Compositor/lib/Nexus/CMakeLists.txt
index 9796858..04da63a 100644
--- a/Compositor/lib/Nexus/CMakeLists.txt
+++ b/Compositor/lib/Nexus/CMakeLists.txt
@@ -26,11 +26,17 @@ set_target_properties(${TARGET} PROPERTIES
 
 if(NEXUS_SERVER_EXTERNAL)
     find_package(NXCLIENT REQUIRED)
+    find_package(NXSERVER REQUIRED)
+
     target_link_libraries(${TARGET}
         PRIVATE
             ${NAMESPACE}nexusserver::${NAMESPACE}nexusserver
             NXCLIENT::NXCLIENT
         )
+    target_include_directories(${TARGET}
+        PRIVATE
+            ${LIBNXSERVER_INCLUDE_DIR}
+        )
 else()
     find_package(NXCLIENT_LOCAL REQUIRED)
     target_compile_definitions(${TARGET}
diff --git a/Compositor/lib/Nexus/NexusServer/CMakeLists.txt b/Compositor/lib/Nexus/NexusServer/CMakeLists.txt
index 8f8c9c9..268474e 100644
--- a/Compositor/lib/Nexus/NexusServer/CMakeLists.txt
+++ b/Compositor/lib/Nexus/NexusServer/CMakeLists.txt
@@ -22,6 +22,8 @@ add_library(${TARGET} STATIC
 
 if (NEXUS_SERVER_EXTERNAL)
     find_package(NXCLIENT REQUIRED)
+    find_package(NXSERVER REQUIRED)
+
     target_link_libraries(${TARGET}
         PRIVATE
             ${NAMESPACE}Core::${NAMESPACE}Core
@@ -30,6 +32,9 @@ if (NEXUS_SERVER_EXTERNAL)
     target_compile_definitions(${TARGET} 
         PRIVATE 
             NEXUS_SERVER_EXTERNAL)
+    target_include_directories(${TARGET}
+        PRIVATE
+            ${LIBNXSERVER_INCLUDE_DIR})
 else()
     find_package(NXSERVER REQUIRED)
     find_package(NEXUS REQUIRED)
